setup.cfg

```diff
@@ -25,6 +25,60 @@ markers =
 inputs = stable_baselines3
 disable = pyi-error
 
+[mypy]
```

Comment:
did you had to add each file manually :see_no_evil: ?

Reply:
Worse than that. First I added all the files, then for each ignored file I checked if removing it from the ignore list really produced a mypy error.
And I deleted the ones that didn't cause an error. 😮‍💨

Reply:
oh boy... thanks for the effort =)